src/aura/Work_Items_List/Work_Items_ListController.js

```diff
@@ -1,4 +1,20 @@
 ({
-	myAction : function(component, event, helper) {
+	doInit : function(component, event, helper) {
+		var statusType = component.get('v.workItemsStatus');
+
+		var action = component.get('c.getWorkItemsByStatusInLast30Days');
+		action.setParam('status', statusType);
+
+		action.setCallback(this, function(response){
+			var state = response.getState();
+			if(state == 'SUCCESS'){
+				var items = response.getReturnValue();
+				component.set('v.workItemsList', items);
+			}else{
+				// no else yet
+				console.log('Failed')
+			}
```

Comment:
I would recommend you to move this code to the helper of the component. It's a salesforce goodPractice in order to maintain our code as modular as we can.